examples/auth-generate-token/page.md

```diff
@@ -0,0 +1,151 @@
+# Generate an Ably token to authenticate clients
```

Comment:
Should this be "request" rather than "generate" (and generate is for JWT?)

Also "Ably Token" needs to be capitalised throughout (sorry)